src/workers/notifications/commentMention.ts

```diff
@@ -24,6 +24,10 @@ const worker: NotificationWorker = {
     if (!postCtx) {
       return;
     }
+    const authors = [...new Set([postCtx.post.authorId, postCtx.post.scoutId])];
+    if (authors.includes(data.commentMention.mentionedUserId)) {
```

Comment:
Do you still need this with the set?

Reply:
It was not being recognized as an array, but I found a workaround by using `.has` API 🤯